script/protocgen-legacy.sh

```diff
@@ -64,18 +68,23 @@ for dir in $proto_dirs; do
         --grpc-gateway_out=logtostderr=true:. \
         $(find "${dir}" -maxdepth 1 -name '*.proto')
 
-    .cache/bin/protoc \
+    run_ts_proto \
```

Comment:
lets not put this into a script and keep using old bulky way.
it allows to see how codegen actually done.

we're also going to switch using buf in a near future, so this will go away